### DISCUSSION TITLE
Kubernetes 란?

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#44] [Q&A] [answered]
Kubernetes 란?

**Comment by tjdud22** [comment, marked as answer]
쿠버네티스는 컨테이너화된 애플리케이션의 배포, 확장 및 관리를 자동화하는 오픈 소스 시스템 탄력적인 컨테이너 운용을 위한 솔루션, 쿠버네티스는 컨테이너 오케스트레이션 도구 중 가장 널리 사용되는 오픈소스 툴
